xrdsst/controllers/base.py

```diff
@@ -154,7 +154,7 @@ def initialize_basic_config_values(self, security_server, config=None):
 
     @staticmethod
     def log_api_error(api_method, exception):
-        logging.error("Exception calling " + api_method + ": " + str(exception))
+        logging.error("Exception calling %s: %s", api_method, str(exception))
```

Comment:
This is something that eluded my notice, since there are quite many formatting changes in this PR. It currently does no harm. Operator ``%`` operator is essentially dead, printf style format specifiers not. However:

* https://docs.python.org/3/howto/logging.html#logging-variable-data

```
...
the old, %-style of string formatting. This is for backwards compatibility: the logging package pre-dates newer formatting options such as str.format() and string.Template. These newer formatting options are supported, but exploring them is outside the scope of this tutorial: ...
```
seems to suggest it is better to either leave it as is or use the Python 3 formatting options.